packages/network-of-terms-catalog/catalog/queries/search/poolparty.rq

```diff
@@ -45,7 +45,9 @@ WHERE {
         FILTER(LANG(?narrower_prefLabel) = "nl")
     }
     OPTIONAL {
-        ?uri skos:exactMatch ?skos_exactMatch .
+        ?uri skos:related ?related_uri .
+        ?related_uri skos:prefLabel ?related_prefLabel .
+        FILTER(LANG(?related_prefLabel) = "nl")
```

Comment:
Relevant for CHT too, which didn’t have `skos:related` yet.

Reply:
👍🏻